include/tune_quda.h

```diff
@@ -152,21 +152,25 @@ namespace quda {
         ret = true;
       }
 
-      if (!tuneGridDim()) 
+      if (!tuneGridDim())
 	param.grid = dim3((minThreads()+param.block.x-1)/param.block.x, 1, 1);
 
       return ret;
     }
 
     /**
-     * @brief For some reason this can't be queried from the device
-     * properties, so here we set set this.  Based on Table 14 of the
-     * CUDA Programming Guide 10.0 (Technical Specifications per
```

Comment:
We should leave this note in as a reference for the `#else` path

Reply:
That was intentional, but I've restored the reference in a new comment in the else path